core/backend/src/IModelDb.ts

```diff
@@ -721,11 +721,17 @@ export abstract class IModelDb extends IModel {
   public reverseTxns(numOperations: number): IModelStatus {
     return this.nativeDb.reverseTxns(numOperations);
   }
+
   /** @internal */
   public reinstateTxn(): IModelStatus {
     return this.nativeDb.reinstateTxn();
   }
 
+  /** @internal */

```

Comment:
`@internal` because we think nobody except us will ever find this useful?

Reply:
Maybe you want to add a `@public` one to TxnManager?

Reply:
reverseTxns and reinstateTxn are internal...restartTxnSession is certainly less useful than those...should these all be public @kabentley ?

Reply:
@calebmshafer how did this auto-complete with this unresolved conversation?

Reply:
@bbastings those are internal because they are accessed via TxnManager.

Reply:
@pmconne no idea... Must be the "beta" tag they have on it, based on the [description of the feature](https://docs.github.com/en/github/administering-a-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-conversation-resolution-before-merging) it shouldn't have.

I'll report the issue to GitHub.

Reply:
> @bbastings those are internal because they are accessed via TxnManager.

I assume so...

The auto-complete was definitely weird...